stratum/hal/lib/tdi/es2k/es2k.bzl

```diff
@@ -0,0 +1,35 @@
+# Copyright 2018-present Open Networking Foundation
+# Copyright 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+# This Starlark rule imports the ES2K SDE shared libraries and headers.
+# The SDE_INSTALL or SDE_INSTALL_TAR environment variable needs to be set;
+# otherwise, the Stratum rules for ES2K platforms cannot be built.
+
+def _impl(repository_ctx):
+    if ("SDE_INSTALL" not in repository_ctx.os.environ and
+        "SDE_INSTALL_TAR" not in repository_ctx.os.environ):
+        print("SDE_INSTALL_TAR is not defined")
+        repository_ctx.file("BUILD", "")
+        return
+    local_install_path = "es2k-bin"
+    if "SDE_INSTALL_TAR" in repository_ctx.os.environ:
+        sde_install_tar_path = repository_ctx.os.environ["SDE_INSTALL_TAR"]
+        repository_ctx.extract(sde_install_tar_path, local_install_path)
+    elif "SDE_INSTALL" in repository_ctx.os.environ:
+        sde_install_path = repository_ctx.os.environ["SDE_INSTALL"]
+        repository_ctx.symlink(sde_install_path, local_install_path)
+    ver = repository_ctx.read(local_install_path + "/share/VERSION").strip("\n")
+    repository_ctx.template(
+        "BUILD",
+        Label("@//bazel:external/es2k.BUILD"),
```

Comment:
We're missing a file (`bazel/external/build/es2k.BUILD`) that defines the SDE libraries and header files.